index.bs

```diff
@@ -405,13 +411,11 @@ When this method is invoked, the user agent MUST execute the following algorithm
     an <a link-for='web'>opaque origin</a>, reject |promise| with a <a>DOMException</a> whose name is "NotAllowedError", and
     terminate this algorithm. Otherwise,
     - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is not <a>present</a>, then set |rpId| to |callerOrigin|, 
-        and |rpIdHash| to the SHA-256 hash of |rpId|.
-    - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is <a>present</a>, then invoke the procedure used for 
-        <a>relaxing the same-origin
-        restriction</a> by setting the `document.domain` attribute, using {{ScopedCredentialOptions/rpId}} as the given value
-        but without changing the current document's `domain`. If no errors are thrown, set |rpId| to the value of `host` as
-        computed by this procedure, and |rpIdHash| to the SHA-256 hash of the UTF-8 encoding of |rpId|. Otherwise, reject |promise| with a
-        <a>DOMException</a> whose name is "SecurityError", and terminate this algorithm.
+        and |rpIdHash| to the SHA-256 hash of the UTF-8 encoding of |rpId|.
+    - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is <a>present</a>, then invoke the [[#algo-relax-same-origin-restriction]],
+        using {{ScopedCredentialOptions/rpId}}. If no errors are thrown, set |rpId| to the value of `host` as
+        computed by this algorithm, and |rpIdHash| to the SHA-256 hash of the UTF-8 encoding of |rpId|. Otherwise, reject
+        |promise| with a <a>DOMException</a> whose name is "{{SecurityError}}", and terminate this algorithm.
```

Comment:
> So this does not propagate out the errors from the relax-same-origin-restriction algorithm?

A learning-curve oversight, thanks for catching it. 

How about this as a rewrite of the last sentence from above:
> Otherwise, reject |promise| with any <a>DOMException</a> thrown by [[#algo-relax-same-origin-restriction]] and terminate these steps.

..?